### PR TITLE
Feature/mkk 18 router employee add dialog

### PR DESCRIPTION
**Pull request by kendrickcarrolldc**
Add dialog form is now creating employees and adding to the database.

### CODE REVIEW

File: client/app/sidenav/employee/add.employee.dialog.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+				<md-input-container class="md-block" flex="50">
+					<label>Last Name</label>
+					<input required name="userName" ng-model="vm.employeeName.last_name">
+				</md-input-container>
+			</div>
+
+			<div layout-padding layout="row">
+				<md-input-container class="md-block" flex="50">
+					<label>Client</label>
```
Comment (kathrynlg) [inline client/app/sidenav/employee/add.employee.dialog.html:26]: this should return an array not a string

File: client/app/sidenav/employee/add.employee.dialog.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+					<input required name="userName" ng-model="vm.employeeName.client">
+
+				</md-input-container>
+
+				<md-input-container class="md-block" flex="50">
+					<label>Skill</label>
+					<input required name="userName" ng-model="vm.employeeName.skill">
```
Comment (kathrynlg) [inline client/app/sidenav/employee/add.employee.dialog.html]: this should return an array not a string

File: client/app/sidenav/employee/employee.controller.js [hunk 3]
```diff
@@ -40,10 +67,11 @@
 			}
 		}
 		
-		$scope.employeeList = EmployeeService.query();
+		$scope.employeeList = EmployeeFact.employeeList;
```
Comment (kathrynlg) [inline client/app/sidenav/employee/employee.controller.js]: you should use this and not scope. replace $scope with this and add the controller (this.controller.employeeList)

**Review by kathrynlg — COMMENTED** [review]
fix the changes I made comments on

**Comment by kendrickcarrolldc** [comment]
Made changes from Donovan's review of my code. 

-changed $scope to this
-changed client and skills to arrays

**Review by kathrynlg — APPROVED** [review]

**Comment by kathrynlg** [comment]
looks good

**Review by lchapoy — APPROVED** [review]

**Pull request merged**